gaps/gap-25_golem_certificates/gap-25_golem_certificates.md

```diff
@@ -30,46 +31,68 @@ When designing certificates structure that will allow to meet requirements menti
 - KYC
 - Inbound network traffic
 
-This means that certificates will be used to sign different kinds of permissions. In case of transferring certification to other companies or signing legal Agreements, we need to ensure that we keep control which permissions they will be able to use.
+This means that certificates will be used to sign different kinds of permissions. In case of transferring certification to other companies or signing legal Agreements, we need to ensure that we control which permissions they will be able to use.
 
-In the future of Golem Network we expect that new, independent features will be implemented by community and they may need to use certification. This document aims to standardize this approach.
+In the future of Golem Network we expect that new, independent features will be implemented by the community and they may need to use certification. This document aims to standardise this approach.
 
 ## Specification
 
-We decided to use [x509 arbitrary extensions](https://www.openssl.org/docs/man1.1.1/man5/x509v3_config.html) to enrich certificates with custom fields describing permissions.
+We decided to use [X.509 Certificate Extensions (Section 4.2)](https://www.rfc-editor.org/rfc/inline-errata/rfc5280.html) to control key usage and permissions in the Golem network.
+
+### Key usage
+
+A certificate can be used to extend the chain of trust via signing other certificates or sign other data that is relevant to the network (for example a Computation Manifest).
+To control what a certificate can sign, standard extensions defined in [RFC 5280 Section 4.2.1](https://www.rfc-editor.org/rfc/inline-errata/rfc5280.html) will be used.
+
+Extensions
+- Key Usage (OID: joint-iso-ccitt(2) ds(5) 29 15) extension must be marked as critical
+-- Bit `digitalSignature` (0) is used to denote the ability to sign data but not certificates, this is used to grant permission to the signed data.
```

Comment:
for me, this sounds a bit awkward. Maybe we could use one of the two variants:

The `digitalSignature` bit (0) ... 

or 

Bit 0 - `digitalSignature` - ...